src/main/groovy/nebula/plugin/publishing/publications/SourceJarPlugin.groovy

```diff
@@ -46,33 +56,13 @@ class SourceJarPlugin implements Plugin<Project> {
                 }
             })
 
-
-            project.plugins.withType(MavenPublishPlugin) {
-                project.plugins.apply(MavenBasePublishPlugin)
-
-                project.publishing {
-                    publications {
-                        nebula(MavenPublication) {
-                            artifact project.tasks.sourceJar
-                        }
-                    }
-                }
-            }
-
-            project.plugins.withType(IvyPublishPlugin) {
-                project.plugins.apply(IvyBasePublishPlugin)
-
-                project.publishing {
-                    publications {
-                        nebulaIvy(IvyPublication) {
-                            artifact(project.tasks.sourceJar) {
-                                type 'sources'
-                                conf 'sources'
-                            }
-                        }
-                    }
-                }
-            }
+            TaskContainer tasks = project.getTasks()
```

Comment:
This is basically `java.withSourceJar()` content except it is using our task so we can avoid breaking customers.